Documentation/ceph-object-bucket-claim-crd.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Object Bucket Claim CRD
+weight: 2800
+indent: true
+---
+
+# Ceph Object Bucket Claim CRD
+
+Rook allows creation and customization of object bucket claim through the custom resource definitions (CRDs). The following settings are available for Ceph Object Bucket Claim. It also requires storage class
+
+## Sample
+##### Stoarge class
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: rook-ceph-bucket
+provisioner: ceph.rook.io/bucket
+# set the reclaim policy to delete the bucket and all objects
+# when its OBC is deleted.
+reclaimPolicy: Delete
+parameters:
+  objectStoreName: my-store
+  objectStoreNamespace: rook-ceph
+  region: us-east-1
+  #bucketName:
+```
+##### Object Bucket Claim 
+```yamlapiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: ceph-bucket
+spec:
+  #bucketName: 
```

Comment:
can be deleted.

Reply:
Sorry I didn't understand this completely.  If u mentioned removing bucketname, I thought it is appropriate to list all options in samples and explained them post that